horovod/spark/torch/remote.py

```diff
@@ -237,9 +237,9 @@ def save_checkpoint():
                                     **reader_factory_kwargs) \
                     if should_validate else empty_batch_reader() as val_reader:
 
-                    train_loader = DataLoader(train_reader,
-                                              batch_size=batch_size,
-                                              shuffling_queue_capacity=shuffle_buffer_size)
+                    train_loader = BatchedDataLoader(train_reader,
```

Comment:
Is BatchedDataLoader strictly better than DataLoader in all cases?  Are there scenarios where DataLoader will perform better than BatchedDataLoader?

Reply:
From my experiments it is strictly better, the only reason we still have DataLoader is that certain data types (string/decimals) are not supported by BatchedDataLoader.